libs/ui/src/enum.ts

```diff
@@ -82,6 +82,8 @@ export enum UI {
   // Colors
   COLOR_WHITE = '--cow-color-white',
   COLOR_BLUE = '--cow-color-blue',
+  COLOR_BLUE_300_PRIMARY = '--cow-color-blue-300-primary',
```

Comment:
Going to use this color coding going forward, as it matches the internal palette being used for marketing. The idea being we'd work on consolidating existing colors into the palette (color coded) colors, as much as possible.

Reply:
Cool! Thanks!